gen3-integration-tests/test_data/test_setup/user.yaml

```diff
@@ -0,0 +1,874 @@
+        - 'peregrine_reader'
+        - 'sheepdog_reader'
+      resource_paths:
+        - '/abc/programs/test_program/projects/test_project2'
+
+    - id: 'abc.programs.test_program2.projects.test_project3-viewer'
+      description: ''
+      role_ids:
+        - 'guppy_reader'
+        - 'fence_reader'
+        - 'peregrine_reader'
+        - 'sheepdog_reader'
+      resource_paths:
+        - '/abc/programs/test_program2/projects/test_project3'
+
+    - id: 'open_data_reader'
+      description: ''
+      role_ids:
+        - 'guppy_reader'
+        - 'fence_reader'
+        - 'peregrine_reader'
+        - 'sheepdog_reader'
+      resource_paths:
+        - '/open'
+        - '/programs/DEV/projects/DICOM_test'
+
+    - id: 'services.sheepdog-admin'
+      description: 'CRUD access to programs and projects'
+      role_ids:
+        - 'sheepdog_admin'
+      resource_paths:
+        - '/services/sheepdog/submission/program'
+        - '/services/sheepdog/submission/project'
+
+    # indexd
+    - id: 'indexd_admin'
+      # this only works if indexd.arborist is enabled in manifest!
+      description: 'full access to indexd API'
+      role_ids:
+        - 'indexd_admin'
+      resource_paths:
+        - '/programs'
+        - '/services/indexd/admin'
+    - id: 'indexd_creator'
+      role_ids:
+        - 'indexd_record_creator'
+      resource_paths:
+        - '/programs'
+
+    - id: 'indexd-tester'
+      description: ''
+      role_ids:
+        - 'indexd_record_creator'
+        - 'indexd_record_reader'
+        - 'indexd_record_updater'
+        - 'indexd_delete_record'
+        - 'indexd_storage_reader'
+        - 'indexd_storage_writer'
+      resource_paths: ['/gen3/programs/QA', '/programs/QA']
+
+    - id: 'workflow_admin'
+      description: 'admin access to argo workflow API'
+      resource_paths: ['/services/workflow/argo/admin']
+      role_ids: ['workflow_admin']
+
+    - id: 'argo'
+      description: 'be able to use argo'
+      resource_paths: ['/argo']
+      role_ids: ['argo_user']
+
+    - id : 'requestor_client_credentials_test'
+      description: 'test policy for client credentials requestor requests'
+      role_ids:
+      - 'requestor_creator'
+      - 'requestor_updater'
+      - 'requestor_deleter'
+      resource_paths:
+      - '/requestor_client_credentials_test'
+
+  # Requestor Integration test policies
+    - id: 'requestor_integration_test'
+      description: 'test policy for integration tests'
+      role_ids:
+      - 'workspace_user'
+      - 'requestor_creator'
+      - 'requestor_updater'
+      - 'requestor_deleter'
+      resource_paths:
+      - '/requestor_integration_test'
+
+
+  resources:
+    # General Access
+    - name: 'data_file'
+      description: 'data files, stored in S3'
+    - name: 'requestor_client_credentials_test'
+      description: 'test resource for client credentials requestor requests'
+    - name: 'cedar'
+      description: 'commons /cedar'
+    - name: 'kayako'
```

Comment:
Kayako has been deprecated and removed from Gen3, so this should is likely no longer used or needed.